### PR TITLE
Fix/pa11y failing tests

### PR DESCRIPTION
**Pull request by mthege**

### CODE REVIEW

File: public/sass/custom/kth-klaro.scss [hunk 1]
```diff
@@ -58,96 +58,95 @@
...
-        max-width: 554px;
-        background-color: $white;
-        border-radius: 0.25rem;
-        box-shadow: 0 2px 10px 2px rgb(0 0 0 / 20%), 5px 5px 10px 0 rgb(0 0 0 / 19%);
-
-        .cn-ok {
-          flex-flow: column !important;
+      font-size: 1rem;
+      padding: 1.5rem;
+      margin: 10px;
```
Comment (falric) [inline public/sass/custom/kth-klaro.scss]: I wonder about padding in rem and margin in px. Is there any reason to not use the same unit for them?

File: public/sass/custom/kth-klaro.scss [hunk 2]
```diff
@@ -58,96 +58,95 @@
...
+      padding-top: 0.5rem;
+      width: 554px;
+      height: fit-content;
+      max-width: 554px;
+      background-color: #fff;
```
Comment (falric) [inline public/sass/custom/kth-klaro.scss]: Why not keep $white?

File: public/sass/custom/kth-klaro.scss [hunk 3]
```diff
@@ -58,96 +58,95 @@
...
+      border-radius: 0.25rem;
+      box-shadow: 0 2px 10px 2px rgba(0, 0, 0, 0.2), 5px 5px 10px 0 rgba(0, 0, 0, 0.19);
+      border: none;
+      bottom: 0;
+      right: 0;
+      left: 0;
+      #id-cookie-notice {
+        color: black;
```
Comment (falric) [inline public/sass/custom/kth-klaro.scss]: Sass-variable instead?

File: public/sass/custom/kth-klaro.scss [hunk 4]
```diff
@@ -58,96 +58,95 @@
...
+      }
+      .cn-ok {
+        flex-flow: column !important;
+      }
+      .cm-link {
+        &.cn-learn-more {
+          color: #007fae;
```
Comment (falric) [inline public/sass/custom/kth-klaro.scss]: Sass-variable instead?

File: public/sass/custom/kth-klaro.scss [hunk 5]
```diff
@@ -58,96 +58,95 @@
...
-                  padding: 0.5rem 1.5rem;
-
-                  @media (max-width: 554px) {
-                    margin-bottom: 10px;
-                  }
+            &.cm-btn-success {
+              color: #fff;
+              background-color: #007fae;
+              border-color: #007fae;
+              border: 1px solid rgba(0, 0, 0, 0);
```
Comment (falric) [inline public/sass/custom/kth-klaro.scss]: Sass-variables instead for `color`, `background-color`, `border-color`, and `border`?

Wait, first `border-color` and then a color code in `border` shorthand?

File: public/sass/custom/kth-klaro.scss [hunk 6]
```diff
@@ -58,96 +58,95 @@
...
+              padding: 0.5rem 1.5rem;
+              line-height: 1.5;
+              border-radius: 0.25rem;
+              min-height: 2.75rem;
+              font-weight: bold;
+              font-size: 1rem;
```
Comment (falric) [inline public/sass/custom/kth-klaro.scss:104]: Is `font-size` needed? Same with `line-height` above.

**Review by falric — COMMENTED** [review]
A few questions and suggestions. 😃

File: public/sass/custom/kth-klaro.scss (hunk 6 above)
Comment (mthege) [reply public/sass/custom/kth-klaro.scss:104]: After checking, line-height is not needed, but font-size needs to be set.

**Review by falric — APPROVED** [review]
✨

**Pull request merged**